Realm/RLMObjectStore.mm

```diff
@@ -28,6 +28,7 @@
 #import "RLMRealm_Private.hpp"
 #import "RLMSchema_Private.h"
 #import "RLMUtil.hpp"
+#import "RLMSwiftSupport.h"
```

Comment:
I think sorting alphabetically is silly. Should instead put the privates together and group other similar apis together.